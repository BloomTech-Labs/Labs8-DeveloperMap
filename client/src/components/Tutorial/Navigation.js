import React from 'react';
import styled from 'styled-components';
import nav from '../../images/navsearch.PNG';
import pinch from '../../images/Pinch_zoom.png';

class Navigating extends React.Component {
    switchPage = () => {
        this.props.history.push('/tutorial/usingmarkers');
    }
    render() {
    return (
        <NavGuide>
            <h1>Navigation</h1>
            <p>
                You can use the search bar at the top left of the map to enter a location. The map will pan to the location 
                on the map. 
            </p>
            <img src={nav} alt=""/>
            <p>By clicking and dragging the map you can also pan accross the map, if you are 
                on mobile, you can zoom into a location by using two fingers and spreading them apart 
                in any direction.
            </p>
            <img src={pinch} alt=""/>
            <Next onClick={this.switchPage}>Next</Next>
        </NavGuide>
    );
    }
}

const NavGuide = styled.div`
  padding: 5%;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  p{
      margin: 2%;
      line-height: 1.25;
  }
`;

const Next = styled.button`
  width: 100px;
  border: none;
  border-radius: 25px;
  background-color: rgba(109, 7, 26, .95);
  cursor: pointer;
  outline: none;
  height: 30px;
  color: white;
  box-shadow: 0 4px 2px -2px gray;
  &:hover {
    background-color: rgba(109, 7, 26, .75);
  }
`;

export default Navigating;