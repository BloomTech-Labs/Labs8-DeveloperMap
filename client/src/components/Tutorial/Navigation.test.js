import React from 'react';
import renderer from 'react-test-renderer';
import Navigation from './Navigation';
import { shallow } from 'enzyme';

describe(`navigation Component`, () => {
    it('renders correctly', () => {
        const tree = shallow(
          <Navigation />
        );
        expect(tree).toMatchSnapshot();
      });

});