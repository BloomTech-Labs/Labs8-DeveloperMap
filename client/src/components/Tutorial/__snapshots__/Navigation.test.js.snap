// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation Component renders correctly 1`] = `
<styled.div>
  <h1>
    Navigation
  </h1>
  <p>
    You can use the search bar at the top left of the map to enter a location. The map will pan to the location on the map.
  </p>
  <img
    alt=""
    src="navsearch.PNG"
  />
  <p>
    By clicking and dragging the map you can also pan accross the map, if you are on mobile, you can zoom into a location by using two fingers and spreading them apart in any direction.
  </p>
  <img
    alt=""
    src="Pinch_zoom.png"
  />
  <styled.button
    onClick={[Function]}
  >
    Next
  </styled.button>
</styled.div>
`;
